firmware/controllers/sensors/map.cpp

```diff
@@ -244,6 +266,18 @@ void initMapDecoder(Logging *sharedLogger DECLARE_ENGINE_PARAMETER_SUFFIX) {
 		digitalMapInput->widthListeners.registerCallback((VoidInt) digitalMapWidthCallback, NULL);
 	}
 
+	if (CONFIG(useFixedBaroCorrFromMap)) {
+		// Read initial MAP sensor value and store it for Baro correction.
+		storedInitialBaroPressure = getRawMap(PASS_ENGINE_PARAMETER_SIGNATURE);
+		scheduleMsg(logger, "Get initial baro MAP pressure = %fkPa", storedInitialBaroPressure);
+		// validate if it's within a reasonable range (the engine should not be spinning etc.)
+		storedInitialBaroPressure = validateBaroMap(storedInitialBaroPressure);
+		if (!cisnan(storedInitialBaroPressure))
```

Comment:
complex if-statements should definitely have curly brackets. maybe even any if statement should use curly brackets